### PR TITLE
Allow a broadcastAlert to have divisionId or teamId

### PR DESCRIPTION
**Pull request by dmccraw**

### CODE REVIEW

File: lib/teamsnap.js [hunk 1]
```diff
@@ -686,8 +686,14 @@ exports.saveBroadcastAlert = function(broadcastAlert, callback) {
   if (!this.isItem(broadcastAlert, 'broadcastAlert')) {
     throw new TSArgsError('teamsnap.saveBroadcastAlert', "`type` must be 'broadcastAlert'");
   }
-  if (!broadcastAlert.teamId) {
-    return this.reject('You must provide a team id.', 'teamId', callback);
+  if (broadcastAlert.isLeague) {
+    if (!broadcastAlert.divisionId) {
```
Comment (amberwebb) [inline lib/teamsnap.js]: Is there a v3 update for this? There are no `isLeague` or `divisionId` properties on broadcastAlerts presently.

Comment (amberwebb) [reply lib/teamsnap.js]: I found the v3 PR with the changes and see those properties in the serializer.

Has this been tested with team?

**Comment by amberwebb** [comment]
I am curious of your plan here with deploying the v3 changes. Typically, we do not create a new SDK version on npm until v3 updates are in master.

Looking at the v3 PR, it looks like alerts in the team app are require updates as well.

**Comment by dmccraw** [comment]
We are trying to find the best synchronization here. We do have a big v3 pr and a big league frontend pr and this pr so I would imagine we are doing v3, teamsnap-sdk and then lfe. Just wanted to make sure there wasn't anything obviously wrong with this pr.

**Comment by amberwebb** [comment]
Thanks. The PR looks fine. Has this been tested in the team app?

File: lib/teamsnap.js (hunk 1 above)
Comment (dmccraw) [reply lib/teamsnap.js]: I have tested this on the team app as well as the tests

**Comment by dmccraw** [comment]
@amberwebb So this is ready to go. Apiv3 has been updated as well as testing on the teamside. Can you publish this to npm?

**Pull request merged**